src/main/java/browserview/BrowserComponent.java

```diff
@@ -123,12 +130,25 @@ private ChromeDriverEx createChromeDriver() {
         ChromeDriverEx driver = new ChromeDriverEx(options, isTestChromeDriver);
         WebDriver.Options manage = driver.manage();
         if (!isTestChromeDriver) {
+            setWindowSize(manage);
+            initialiseJNA();
+        }
+        return driver;
+    }
+
+    public void setWindowSize(WebDriver.Options manage) {
```

Comment:
Nah, it's okay. We can just mention it in the comment maybe, with a high-level description.